app/views/broadcasts/form.html.erb

```diff
@@ -2,7 +2,7 @@
 
   <%= top_action_links(@broadcast, :only => :index) %>
 
-  <%= f.field(:recipient_ids, :type => :hidden) %>
+  <%= f.field(:recipient_ids, :type => :hidden, :multiple => true) %>
```

Comment:
Yeah. This basically just adds `[]` to the end of the `<input>` name. I was expecting it to also create a single `<input>` tag per value, but instead it uses a single comma-separated `value`. Hence the changes to `recipient_ids=` in `app/model/broadcast.rb`.